### PR TITLE
Version of db2tcl used by HammerDB including added Win build

### PR DESCRIPTION
**Pull request by sm-shaw**
This is the current version of db2tcl that gets downloaded from www.hammerdb.com when you do a HammerDB build from source (https://www.hammerdb.com/blog/uncategorized/how-to-build-hammerdb-from-source/). This builds without intervention on both Linux and Windows. 
Includes recent additions of Db2 apis to create and drop databases and do force applications.

### CODE REVIEW

**Comment by sm-shaw** [comment]
closes issue #6

File: src/db2tclcmds.c [hunk 1]
```diff
@@ -1,8 +1,9 @@
 /* $Id$ */
 
 #include "db2tclcmds.h"
-
+/*#include <sqle819a.h>*/ /* For sqlecrea(): struct sqledbdesc: sqldbudc sqle_819_500 */
```
Comment (memmertoIBM) [inline src/db2tclcmds.c]: Please remove this line

**Review by memmertoIBM — CHANGES_REQUESTED** [review]
Looks good, other than one minor cleanup.

Can you also update:
AUTHORS - to include the folks from Intel that worked on these changes
README - to mention the new APIs

File: src/db2tclcmds.c (hunk 1 above)
Comment (sm-shaw) [reply src/db2tclcmds.c]: changes have been made

**Review by memmertoIBM — APPROVED** [review]

**Pull request merged**